### ISSUE TITLE
Qwen3-Embedding-0.6B generate_llm_qnn.py 再最新代码中报错

### ISSUE THREAD

**Issue by PencilGo** [alibaba/MNN#4481]
改bug已经被修复 [LLM:Bugfix] Fix Qwen3-Embedding QNN export crash #4216， 但后续提交覆盖了这个修改
1. generate_llm_qnn.py 脚本问题：后续提交覆盖了 commit 5889d95 的修复，使用了错误的 IO 生成流程，总是为 LLM 生成包含 logits_index 输入和 logits 输出的配置，但 embedding 模型使用的是 sentence_embeddings 输出。
2. generateLlmIO.cpp 工具问题：
   - 硬编码 llm.mnn 作为模型文件名，但 embedding 模型使用 embedding.mnn
   - isEmbeddingModel() 函数无法从 llm_config.json 中检测到 embedding 模型（因为缺少 output_names 和 model_type 字段）

**Comment by Qxinyu** [comment]
你好，建议参考 `docs/transformers/llm.md` 中的 **用法三：使用自定义 input_json 转换任意模型** 部分，自己制定输入来转 QNN 模型。这种方式可以灵活指定模型的输入输出配置，避免自动生成流程对 embedding 模型不兼容的问题。
